crates/routes/src/feeds.rs

```diff
@@ -274,7 +273,7 @@ async fn get_feed_community(
   let community = Community::read_from_name(&mut context.pool(), community_name, false)
     .await?
     .ok_or(LemmyErrorType::NotFound)?;
-  if community.visibility != CommunityVisibility::Public {
+  if community.visibility.can_view_without_login() {
```

Comment:
I think this needs a `!` . 

If I *can* view it without a login, no errors should be thrown.

Reply:
Fixed and checked the other uses